bindata/network/ovn-kubernetes/006-ovnkube-config.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# The config file used by ovnkube is passed as a configmap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: ovnkube-config
+  namespace: openshift-ovn-kubernetes
+data:
+  config:   |-
+    [default]
+    mtu="{{.MTU}}"
+
+    [kubernetes]
+    service-cidr="{{.OVN_service_cidr}}"
+    cluster-subnets={{.K8S_cluster_subnets}}
+    ovn-config-namespace="{{.OVN_namespace}}"
```

Comment:
@pecameron For the namespace comment, I actually meant, can't we just do:

`ovn-config-namespace=openshift-ovn-kubernetes`

here and we're done? No need to template that if we know it's always going to be the same thing for OpenShift.

Reply:
It come up in both configmaps. Having it once here keeps things in sync.

Reply:
@pecameron Ok, that's fine then. When Numan's HA stuff lands we'll remove anything to do with endpoints from ovnkube.sh and then we can consolidate.